src/foam/nanos/crunch/UserReferencedPayload.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * @license
+ * Copyright 2021 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+ foam.CLASS({
+  package: 'foam.nanos.crunch',
+  name: 'UserReferencedPayload',
+  documentation: 'Capability user related payload class - attaches listeners to wizardlets if defined',
+
+  imports: [
+    'wizardlets'
+  ],
+
+  properties: [
+    {
+      class: 'StringArray',
+      name: 'wizardletsToListen',
+      visibility: 'HIDDEN',
+      documentation: `
+        (Capability IDs)
+        Select wizardlets to listen on changes based on the capability defined on wizardlet.
+      `
+    }
+  ],
+
+  methods: [
+    function init() {
+      // Initialize listeners on defined wizardlets in reference to their capability ids
+      if ( this.wizardlets ) {
+        this.wizardletsToListen.forEach(capabilityId => {
+          wizardlet = this.wizardlets.find(wizardlet => wizardlet.capability.id === capabilityId );
```

Comment:
In the case of MinMax, if choice A has prereq C and choice B has prereq C. There will be two different wizardlets instantiated which use prereq C so not sure if you tested this on a minmax might have some unexpected behaviour. But from what I read this is an opt-in feature correct?

Reply:
I'll have to take some time to review this. From what I was aware of duplicate prerequisites are combined and a unique prerequisites list is displayed in the wizard.

Reply:
Duplicate prerequisites of minmax capabilities aren't combined... oddly enough - should be following suit to how prerequisites of non minmax capabilities are unique when presented in the wizard.

Anyways, the intended behaviour of applying a listener on the wizardlet is as expected. When a wizardlet with a sub listener changes, send a pub to any payload listening to those changes. In the case of the minmax duplicates, the payloads are listening to the change events.